docker/tpch/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM debian:latest
+
+LABEL maintainer="tjveil@gmail.com"
```

Comment:
TODO would need to ask Tim if he wants to be the maintainer of this dockerfile, if it were to merge into master